descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Module KB EBSCO Module",
+  "provides": [
+    {
+      "id": "kb-ebsco-java",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages"
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/eholdings/packages"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages/{packageId}/resources"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/providers"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/providers/{provider_id}"
```

Comment:
Can we please maintain consistency here: Few endpoints have camelcase like "packageId" where as others have snake case like "provider_id"? I know, its our bad when we put RAML together for mod-kb-ebsco since multiple developers worked on putting together that RAML file. Thanks!

Reply:
@sduvvuri-ebsco will rename it. Thanks